src/InterNations/Component/Solr/Expression/ExpressionBuilder.php

```diff
@@ -403,12 +404,13 @@ public function date(DateTime $date = null, $timezone = false)
      * Create a range between two dates (one side may be unlimited which is indicated by passing null)
      *
      * @param DateTime $from
+     * @param bool $inclusiveFrom
      * @param DateTime $to
-     * @param boolean $inclusive
+     * @param bool $inclusiveTo
      * @param boolean $timezone
      * @return ExpressionInterface|null
      */
-    public function dateRange(DateTime $from = null, DateTime $to = null, $inclusive = true, $timezone = false)
+    public function dateRange(DateTime $from = null, DateTime $to = null, $inclusiveFrom = true, $inclusiveTo = false, $timezone = false)
```

Comment:
This would cause a BC break. Do you plan on releasing a new version with BC breaks anytime soon, @lstrojny ?